src/main/java/step2/domain/Number.java

```diff
@@ -0,0 +1,85 @@
+package step2.domain;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+/*
+ * LOTTO GAME
+ * Number
+ * ver. 1.0
+ * 2020.05.31
+ * Copyright ...
+ */
+public class Number {
+
+    private static final Pattern patternNumbersOnly = Pattern.compile("^[-+]?[0-9]+$");
+
+    public static void checkNumber(String inputString) {
+        if (!patternNumbersOnly.matcher(inputString).matches()) {
+            throw new IllegalArgumentException("Found a not number(s).");
+        }
+    }
+
+    public static void checkNumber(String[] inputString) {
+        Arrays.stream(inputString)
+                .forEach(string -> {
+                            if (!patternNumbersOnly.matcher(string).matches()) {
+                                throw new IllegalArgumentException("Found a not number(s).");
+                            }
+                        }
+                );
+    }
+
+    public static void checkNotNumber(int number) {
+
+        if (number < 0) {
+            throw new IllegalArgumentException("Found a Illegal Argument(s).");
+        }
+
+    }
+
+    public static void checkNotNumber(String number) {
+
+        if (Objects.isNull(number) || number.isEmpty()) {
+            throw new IllegalArgumentException("Found a Illegal Argument(s).");
+        }
+
+        if (Integer.valueOf(number) < 0) {
+            throw new IllegalArgumentException("Found a Illegal Argument(s).");
+        }
+
+    }
+
+    public static void checkNotNumber(String[] numbers) {
+
+        // find a negative number(s).
+        if (Arrays.stream(numbers)
+                .map(Integer::parseInt)
+                .filter(number -> number < 0)
+                .count() > 0) {
```

Comment:
```suggestion
        int count = Arrays.stream(numbers)
                .map(Integer::parseInt)
                .filter(number -> number < 0)
                .count();
        if (count > 0) {
```
위와 같이 지역변수로 할당하여 연산하는것은 어떨까요 ?

Reply:
네 식이 너무 길면 분리 하겟습니다.